src/cloud-api-adaptor/test/e2e/common.go

```diff
@@ -558,6 +558,7 @@ type CloudAssert interface {
 	HasPodVM(t *testing.T, id string)                             // Assert there is a PodVM with `id`.
 	GetInstanceType(t *testing.T, podName string) (string, error) // Get Instance Type of PodVM
 	DefaultTimeout() time.Duration                                // Default timeout for cloud operations
+	GetPodVMResource(t *testing.T, id string) (uint, uint, error) // Get CPU and Memory resource of PodVM
```

Comment:
Can we refactor this with the `GetInstanceType` logic somehow as they are doing the same thing?